README.rst

```diff
@@ -45,7 +45,8 @@ api/*
   utility and helper functions to encode/decode
   to/from OCF’s data model, module for encoding and interpreting type 4
   UUIDs, base64 strings, OCF endpoints, and handlers for the discovery, platform
-  and device resources.
+  and device resources. A complete reference of the Iotivity-Lite API can be found
+  `here <https://cascoda.github.io/iotivity-lite-doxygen/>`_.
```

Comment:
It would be nice for the description of the URL to be bigger - I missed it on my first read of the README!

```suggestion
  and device resources. `A complete reference of the Iotivity-Lite API can be found
  here <https://cascoda.github.io/iotivity-lite-doxygen/>`_.
```